scripts/validate-version-bump.py

```diff
@@ -0,0 +1,65 @@
+#!python
```

Comment:
My only comment is to rename this to `validate_version_bump.py` so it is `lower_snake_case` like the other script files in this directoryl.

Reply:
Change incorporated, though to be fair, `validate-version-bump.sh` predated `run_tests.sh`.a